frontend/tests/services/featureFlags/featureFlagHelpers.test.ts

```diff
@@ -58,6 +58,7 @@ describe("getFeatureFlagsFromCookie", () => {
   });
 
   // // do we still need to support this?
+  // eslint-disable-next-line jest/no-disabled-tests, jest/no-commented-out-tests
```

Comment:
I think I've answered by question here at this point. We can probably delete this test safely